test/integration/replicaset/replicaset_test.go

```diff
@@ -607,7 +596,7 @@ func TestPodOrphaningAndAdoptionWhenLabelsChange(t *testing.T) {
 	rs := newRS("rs", ns.Name, 1)
 	rs.Spec.Selector.MatchLabels = labelMap
 	rs.Spec.Template.Labels = labelMap
-	createRSsPods(t, c, []*v1beta1.ReplicaSet{rs}, []*v1.Pod{}, ns.Name)
+	rss, _ := createRSsPods(t, c, []*v1beta1.ReplicaSet{rs}, []*v1.Pod{}, ns.Name)
```

Comment:
@enisoc 
I changed `TestPodOrphaningAndAdoptionWhenLabelsChange` based on your feedback. Do you mind to have another review for just the test before i worked on other tests? 
Ignore other parts of the codes. I commented out other tests to facilitate testing of individual test, so the codes are messy (although still able to be compiled). I use `bazel test //...`, but am still looking for the regex option that filters test name.